config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -1,14 +1,14 @@
-# periodic jobs
-
+# AUTOGENERATED by https://git.k8s.io/kubeadm/kinder/ci/tools/update-workflows
 periodics:
+
 - name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-21-latest
   interval: 2h
   decorate: true
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
   annotations:
-    testgrid-dashboards: sig-cluster-lifecycle-kubeadm,sig-release-master-informing
+    testgrid-dashboards: sig-cluster-lifecycle-kubeadm, sig-release-master-informing
```

Comment:
Just for double checking, Is the additional space here tolerated by test grid machinery?

Reply:
yes